src/course/handbook/course-rules.md

```diff
@@ -30,11 +30,11 @@ If someone does point out that you broke a rule, try not to take it personally.
 
 ## Respect each other
 
-Please be respectful of your peers on the course, the staff team, and the wider community
+Please be respectful of your peers on the course, the core team, and the wider community
 
-Respect others' time—make sure you arrive on time for everything on the schedule. This means being ready _before_ the scheduled start time, not off making coffee. Also please let your Course Facilitator and cohort know if you're going to be absent.
+Respect each others' time—make sure you arrive on time for everything on the schedule. This means being ready _before_ the scheduled start time, not off making coffee. Also please let your course facilitator and cohort know if you're going to be absent.
```

Comment:
@yvonne-liu this changes the meaning slightly. I meant "respect everyone involved's time (mentors, core team, speakers etc)", not just "each other" (other cohort members)